### DISCUSSION TITLE
Playwright Smoke Test Only Loads First URL – Need Clarification or Updated Example

### DISCUSSION THREAD

**Discussion by jared-christensen** [artilleryio/artillery#3367] [Artillery]
I’m new to load testing and have been trying to follow [Example 2: A smoke test](https://github.com/artilleryio/artillery/tree/main/examples/browser-load-testing-playwright#example-2-a-smoke-test) from the browser load testing section in the Artillery repo. I cloned it and ran it, but it only processes the first URL in the pages.csv file, even though there are multiple URLs listed.

At this point, I’m not sure if the example is outdated, if I’m missing something, or maybe I’m just not fully getting the concept. Is there a better or more up-to-date example I should be looking at for smoke testing with Playwright?

Any guidance or direction would be really helpful!

**Comment by hassy** [comment]
@jared-christensen thanks for flagging this up, will update the example shortly!

**Comment by hassy** [comment]
@jared-christensen updated the example, it should work as expected now. Thanks again!

**Comment by jared-christensen** [comment]
Awesome, thanks so much for the quick update! 🎉 I’ll pull the latest and give it another go. Appreciate the help, @hassy!

**Comment by jared-christensen** [comment]
I just tried the updated example, and it works great! Thanks so much for the quick fix!
